static/js/components/ClassificationForm.jsx

```diff
@@ -0,0 +1,207 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-danger */
+
+import React, { useReducer } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import Select from "@material-ui/core/Select";
+import InputLabel from "@material-ui/core/InputLabel";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import FormControl from "@material-ui/core/FormControl";
+import MenuItem from "@material-ui/core/MenuItem";
+import { makeStyles } from "@material-ui/core/styles";
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import * as Actions from '../ducks/source';
+
+
+function makeMenuItem(taxonomy, index) {
+  const render_string = `${taxonomy.name} (${taxonomy.version})`;
+
+  return (
+    <MenuItem value={index} key={index.toString()}>
+      {render_string}
+    </MenuItem>
+  );
+}
+
+
+const ClassificationForm = ({ obj_id, taxonomyList }) => {
+  const latest_taxonomyList = taxonomyList.filter((t) => t.isLatest);
+
+  function reducer(state, action) {
+    switch (action.name) {
+      case 'taxonomy_index':
+        return {
+          ...state,
+          [action.name]: action.value,
+          allowed_classes: latest_taxonomyList[action.value].allowed_classes,
+        };
+      default:
+        return {
+          ...state,
+          [action.name]: action.value
+        };
+    }
+  }
+
+  const submitDispatch = useDispatch();
+
+  const initialState = {
+    taxonomy_index: latest_taxonomyList.length > 0 ? 0 : null,
+    classification: null,
+    probability: 1.0,
+    class_select_enabled: false,
+    probability_select_enabled: false,
+    probability_errored: false,
+    allowed_classes: latest_taxonomyList.length > 0 ?
+      latest_taxonomyList[0].allowed_classes : [null]
+  };
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const { handleSubmit } = useForm();
```

Comment:
react-hook-form is not actually ever being used below, so can remove this and change the form's `onSubmit` below

Reply:
done.

Reply:
Looks like this still needs to be removed